src/ResourceManager/Insights/Commands.Insights/help/Add-AzureRmLogProfile.md

```diff
@@ -106,7 +117,11 @@ Accept wildcard characters: False
 ```
 
 ### -StorageAccountId
-Specifies the ID of the Storage account.
+Specifies the ID of the Storage account. ID is the fully qualified Resource ID of the storage account for example  
+```yaml
```

Comment:
@angshumannayak if the yaml block you added is not removed, then `platyPS` will throw an error when it tries to convert this markdown file in MAML, which affects the resulting help the user sees when `Get-Help` is ran. The content you have added can remain, it just cannot exist within a yaml block.